jupyter/Notebook Binding/Pass Rate.ipynb

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
Now that I look closer at this, this appears to be just a direct copy of our Pass Rate notebook. Should this be checked in here?